index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link href="https://fonts.googleapis.com/css?family=Poppins" rel="stylesheet">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+    <main>
+        <section class="container">
+                <nav class="nav_bar">
+                        <span class="myLogo"><a href="#"><img src="image/My_Logo.svg" alt="My Logo"></a></span>
+                        <span class="contact"><a href="#"><img src="image/Icon - Menu.svg" alt="Contact Us"></a></span>
+                </nav>
+            <article class="img_background">
+                    <h1 class="header" >I’m Mariane<br> 
+                        Glad to see<br> you!</h1>
+                    <p class="content">I’m a software developer! I can help you build a product , 
+                        feature or website Look through some of my work and experience! If you like 
+                        what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <footer>
+                <nav class="connect"><a href="#"><img src="image/Social Media.svg" alt="Let's Connect"></a></nav>
```

Comment:
[Optional] I think that the `Let's Connect` shouldn't be enclosed in a `nav` tag because the nav tag is semantically meant to hold a set of navigation links but we have only one anchor tag, so that's kind of redundant.